pubspec.yaml

```diff
@@ -1,12 +1,12 @@
 name: mockito
-version: 2.2.2
+version: 2.2.3
 authors:
   - Dmitriy Fibulwinter <fibulwinter@gmail.com>
   - Dart Team <misc@dartlang.org>
 description: A mock framework inspired by Mockito.
 homepage: https://github.com/dart-lang/mockito
 environment:
-  sdk: '>=1.21.0 <2.0.0'
+  sdk: '>=2.0.0-dev.16.0 <2.0.0'
```

Comment:
This is further backwards compatible than this...supposedly as far back as 1.24.0. However, its possible there were still bugs at that time. I'll do some experiments, otherwise I think the safest restriction is dev.16

Reply:
OK I'll leave it at 1.24.0; no features, just a compatibility release.